api/api_full.go

```diff
@@ -906,6 +906,28 @@ type FullNode interface {
 
 	RaftState(ctx context.Context) (*RaftStateData, error) //perm:read
 	RaftLeader(ctx context.Context) (peer.ID, error)       //perm:read
+
+	// Actor events
+
+	// GetActorEvents returns all FVM and built-in Actor events that match the given filter.
```

Comment:
```suggestion
	// GetActorEvents returns all FEVM user actor and built-in Actor events that match the given filter.
```

(technically everything on filecoin is executed by FVM 😄 i call this a side effect of too-good-of-branding 😸

Reply:
"user-programmed and built-in actor events"